### PR TITLE
LPS-159991 | master

### PR DESCRIPTION
**Pull request by sontruongces**
Hi team,

This is the fix for ticket [LPS-159991](https://issues.liferay.com/browse/LPS-159991). The original PR has passed my review at sontruongces#27.
Please help me to review and give feedback.

Thanks!
cc: @quanhuynhces

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by liuxingye** [comment]
ci:test:relevant

File: modules/apps/contacts/contacts-web/src/main/java/com/liferay/contacts/web/internal/portlet/ContactsCenterPortlet.java [hunk 1]
```diff
@@ -712,6 +714,12 @@ public void updateSocialRequest(
 		}
 
 		writeJSON(actionRequest, actionResponse, jsonObject);
+
+		long userNotificationEventId = ParamUtil.getLong(
+			actionRequest, "userNotificationEventId");
+
+		_updateArchived(
+			themeDisplay.getUserId(), userNotificationEventId, true);
```
Comment (adolfopa) [inline modules/apps/contacts/contacts-web/src/main/java/com/liferay/contacts/web/internal/portlet/ContactsCenterPortlet.java:722]: Why the additional `archived` parameter? It is always set to `true`. Can you remove that param?

Also, can you check the value of `userNotificationEventId`, and only call `_updateArchived` if it is not 0?

Finally, shouldn`t this code be moved just before `jsonObject.put("success", Boolean.TRUE);`? As a user, I'd prefer for the notification not to be marked as read if the operation failed.

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by adolfopa** [comment]
@sontruongces See my suggestions.

File: modules/apps/contacts/contacts-web/src/main/java/com/liferay/contacts/web/internal/portlet/ContactsCenterPortlet.java (hunk 1 above)
Comment (sontruongces) [reply modules/apps/contacts/contacts-web/src/main/java/com/liferay/contacts/web/internal/portlet/ContactsCenterPortlet.java]: @adolfopa The code was updated based on your feedback by @quanhuynhces . Thanks!

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by liuxingye** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:reevaluate:1387401_3383

**Comment by adolfopa** [comment]
Just started reviewing :)

**Comment by adolfopa** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/122517

**Pull request closed without merging**